src/GitHub.Api/Git/RepositoryManager.cs

```diff
@@ -9,18 +9,18 @@ namespace GitHub.Unity
     interface IRepositoryManager : IDisposable
     {
         event Action<bool> OnIsBusyChanged;
-
-        event Action OnCommitChanged;
-        event Action OnLocalBranchListChanged;
-        event Action OnRemoteBranchListChanged;
+        event Action<string> OnHeadUpdated;
         event Action<GitStatus> OnStatusUpdated;
-        event Action<ConfigBranch?> OnActiveBranchChanged;
-        event Action<ConfigRemote?> OnActiveRemoteChanged;
```

Comment:
So if the `RepositoryManager` does not have the current state, it cannot determine if the state has changed.